### DISCUSSION TITLE
VizTexture - Preview disappears if checking the pixel values

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#185] [! Welcome - readme first!]
When checking the pixel values in VizTexture, the preview in the op disappears.
(Bug reproduceable in latest versions of Opera and Chrome.)

See attached GIF to reproduce:
![Cables gl VizTexture bug-02](https://user-images.githubusercontent.com/97125854/156663433-8fbc2749-b97a-4317-ae86-220925d6b0f8.gif)

**Comment by franckaubry** [comment]
Also after the bug is triggered, the patch goes into CPU overdrive mode.
![image](https://user-images.githubusercontent.com/97125854/156665345-6b3f9b0e-aefd-405e-a757-c4f30a0f89f1.png)

Link to patch:
https://cables.gl/p/622143b545c0278681d37677?s=ZIQmjFE06eaIawe1

**Comment by pandrr** [comment]
interesting bug, thanks.

probably this spreadsheet view will be removed, the new approach is a viz op for it:
you can already check it out on dev.cables.gl

![Screenshot 2022-03-04 at 09 05 27](https://user-images.githubusercontent.com/98792/156723998-09e2fda0-4993-4ab3-9689-c5280cc3c132.png)

sadly this reading back textures from gpu to cpu is very slow in webgl

**Reply by franckaubry** [reply]
Thanks for looking into it. Glad that VizTextureTable is live! Will try it out :)
